src/main/java/mate/academy/springboot/datajpa/model/dto/ProductResponseDto.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.springboot.datajpa.model.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class ProductResponseDto {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    private String category;
```

Comment:
categoryId

Reply:
for what id?

Reply:
you should return id of category, and then whole category model can be fetched by this id